.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,55 @@
+name: Draft Release & Publish
+on: [workflow_dispatch]
+
+jobs:
+  Release-Artifacts:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    # Checkcout code
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Read cahngelog and read versions etc.
+    - name: Get current release version
+      id: package_version
+      uses: martinbeentjes/npm-get-version-action@v1.1.0
+    - name: Check version is mentioned in Changelog
+      id: changelog_reader
+      uses: mindsers/changelog-reader-action@v2
+      with:
+        validation_depth: 10
+        version: ${{ steps.package_version.outputs.current-version }}
+        path: 'CHANGELOG.md'
+    # Set the DRAFT_VERSION variable
+    - name: Set the draft version value
+      id: step_one
```

Comment:
I don't see this id being referenced anywhere. Typically ids are only added if it will be referenced

Reply:
Early Draft for everyone eyes and concensus. Yep, 100 %, That's the plan this is for everyone's eye and testing, so that all see this and agree.